### DISCUSSION TITLE
GitHub Workflow Status (with event) not Working?

### DISCUSSION THREAD

**Discussion by zouyaoji** [badges/shields#8742] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24430116/208879463-b582afc0-96be-4adb-ab40-32e5ab7d6632.png)

According to my configuration, there should be no need to fill in the branch, right? I have tried both dev and main branches and got wrong results.  
![image](https://user-images.githubusercontent.com/24430116/208879833-d55483db-d3de-41e5-a252-1f59543d507f.png)

https://github.com/zouyaoji/vue-cesium/blob/dev/.github/workflows/publish-npm.yml

**Comment by chris48s** [comment, marked as answer]
This is an example of https://github.com/badges/shields/issues/8736 You should now be able to solve this by omitting the branch param

**Reply by zouyaoji** [reply]
good job!

![image](https://user-images.githubusercontent.com/24430116/209042003-01fe1936-69c5-4440-b4d3-e9fc31f4c597.png)
